code/edgeidentity/build.gradle

```diff
@@ -222,12 +218,7 @@ dependencies {
     androidTestImplementation "androidx.test.ext:junit:${rootProject.ext.junitVersion}"
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
     androidTestImplementation 'com.fasterxml.jackson.core:jackson-databind:2.9.9'
-
-    // TODO: Uncomment next line when Identity 2.0 artifacts are published to maven
-    // implementation "com.adobe.marketing.mobile:identity:2.+"
-    androidTestImplementation ('com.adobe.marketing.mobile:identity:2.0.0-SNAPSHOT') {
-        transitive = false
-    }
+    androidTestImplementation 'com.adobe.marketing.mobile:identity:2.+'
```

Comment:
nit: Should we add identity version to properties and use that for both the app and the integration tests?
`mavenDirectIdentityVersion=2.0.0`

androidTestImplementation 'com.adobe.marketing.mobile:identity:${rootProject. mavenDirectIdentityVersion}'